src/client/app/chat/chat.component.ts

```diff
@@ -255,13 +260,33 @@ export class ChatComponent implements OnInit, AfterViewChecked {
     this.createForm();
     this.receiveMessageFromSocket();
     this.receiveUpdatedMessageFromSocket();
-    //this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
+    this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
+  }
+  
+  //Doctor modal window open methhode
+  openDoctor(doctorModal:any) {
+    this.getDoctors();
+    this.modalService.open(doctorModal, {size: 'lg'}).result.then((result) => {
+      this.closeResult = `Closed with: ${result}`;
+    }, (reason) => {
+      this.closeResult = `Dismissed ${this.getDismissReason(reason)}`;
+    });
+  }
+  
+  private getDismissReason(reason: any): string {
```

Comment:
Method not necessary since we are not interested in the reason modal closes. You may delete this method.